express_relay/examples/easy_lend/README.md

```diff
@@ -0,0 +1,6 @@
+# EasyLend Protocol
+
+EasyLend is a simplified lending protocol that uses Express Relay for avoiding value leakage on liquidations.
+It uses Pyth price feeds to calculate the asset values and the liquidation thresholds.
+
+This project illustrates how to use the Express Relay SDK for contract integration and publishing opportunities.
```

Comment:
let's include some notes about what the `monitor.ts` script is, what it does, how it uses recent pyth prices to assess liquidatability, etc.

Reply:
maybe also worth mentioning some notes around how ezlend works and why it potentially allows undercollateralized vault creation (for testing reasons)